src/components/desktop/desktop.vue

```diff
@@ -43,14 +43,16 @@
     </template>
 
     <template #main-aside>
-      <SelectedFilters
-        v-if="$x.totalResults > 0"
-        class="x-list x-list--gap-05 x-list--padding-05 x-list--padding-bottom"
-      />
-      <Facets
-        v-if="$x.totalResults > 0"
-        class="x-list--gap-06 x-list--padding-05 x-list--padding-top"
-      />
+      <template v-if="hasQuery">
```

Comment:
You can raise the `v-if` to the `<template #main-aside>` tag: `<template v-if="hasQuery" #main-aside>`